Classes/Issues/Comments/IssueCommentSectionController.swift

```diff
@@ -281,7 +281,8 @@ DoubleTappableCellDelegate {
             cell.delegate = self
         }
         
-        if let cell = cell as? DoubleTappableCell {
+        if !(viewController is PullRequestReviewCommentsViewController),
```

Comment:
Ack, strongly discourage reaching into the VC like this. If the VC is renamed, this gets refactored, or used in another surface, it will silently break.

There's already a flag that we use to show reactions or not here:

https://github.com/rnystrom/GitHawk/blob/master/Classes/Issues/Comments/IssueCommentSectionController.swift#L204-L206

Can we just reuse that flag?